src/utils.js

```diff
@@ -1,5 +1,36 @@
 import dayjs from 'dayjs';
-import { Type, Destination, Filter, Sorting } from './enums';
+import { RenderPosition, Type, Destination, Filter, Sorting } from './enums';
+
+const renderTemplate = (container, template, place) => {
+  container.insertAdjacentHTML(place, template);
+};
+
+const render = (container, element, place) => {
+  switch (place) {
+    case RenderPosition.AFTERBEGIN:
+      container.prepend(element);
+      break;
+    case RenderPosition.BEFOREEND:
+      container.append(element);
+      break;
+    case RenderPosition.BEFOREBEGIN:
+      container.parentNode.insertBefore(element, container);
+  }
+};
+
+const renderNestedElement = (container, nestedElement, place) => {
+  if (nestedElement) {
+    render(container, nestedElement, place);
+  }
+};
+
+const createElement = (template) => {
+  const newElement = document.createElement('div');
+
+  newElement.innerHTML = template;
+
+  return newElement.firstChild;
```

Comment:
```suggestion
  return newElement.firstElementChild;
```
Просто `firstChild` считает пробелы за детей 😥

Reply:
Видела, но почему-то просто убрала пробел :) 
Исправила